File: src/core/command_handlers/system.py
import os

class SystemCommands:
    @staticmethod
    def handle_clear(ui):
        os.system("cls" if os.name == "nt" else "clear")
        return True

    @staticmethod
    def handle_exit(chat_manager):
        chat_manager.should_exit = True
        import sys
        sys.exit(0)

File: src_js/core/commandHandlers/system.js
export class SystemCommands {
  static handleClear(ui) {
    process.stdout.write('\x1Bc');
    return true;
  }

  static handleExit(chatManager) {
    chatManager.shouldExit = true;
    console.log('Shutting down...');
    process.exit(0);
  }
}